partitura/io/importmatch.py

```diff
@@ -12,41 +12,16 @@
 from partitura import score
 from partitura.score import Part, Score
 from partitura.performance import PerformedPart, Performance
-from partitura.musicanalysis import estimate_voices, estimate_key
 
 from partitura.io.matchlines_v0 import (
     FROM_MATCHLINE_METHODS as FROM_MATCHLINE_METHODSV0,
-    parse_matchline as parse_matchlinev0,
     MatchInfo as MatchInfoV0,
```

Comment:
Why were these imports removed? Was it because they are not being used?

Reply:
I'm not sure now for these specific lines, but yes I removed several unused imports in importmatch when I added the measure number and id refactor.